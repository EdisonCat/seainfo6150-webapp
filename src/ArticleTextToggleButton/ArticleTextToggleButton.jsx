import React from "react";
import styles from "./ArticleTextToggleButton.module.css";
export default function ArticleTextToggleButton(props) {
	return (
    <button onClick={(e) => props.onClick(e)} className={styles.button}>
        {props.buttonText}
    </button>);
}
