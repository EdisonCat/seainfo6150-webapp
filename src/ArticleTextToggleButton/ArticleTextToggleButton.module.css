.button {
    border-bottom: 2px solid black;
    border-top: 2px solid grey;
    border-radius: 15px;
    width: 100px;
    height: 50px;
    background-color: red;
    color: white;
    cursor: pointer;
}