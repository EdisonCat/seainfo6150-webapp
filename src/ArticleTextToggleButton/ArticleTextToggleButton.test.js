import React from "react";
import ArticleTextToggleButton from "./ArticleTextToggleButton";

describe("ArticleTextToggleButton tests", () => {
  it("component rendered correctly", () => {
    const onClick = jest.fn();
    const { container } = render(<ArticleTextToggleButton buttonText={"Show more"} onClick={onClick}/>);
    expect(container).toMatchSnapshot();
  });
});