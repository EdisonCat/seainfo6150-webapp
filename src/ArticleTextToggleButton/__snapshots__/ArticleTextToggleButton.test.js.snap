// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleTextToggleButton tests component rendered correctly 1`] = `
<div>
  <button
    class="button"
  >
    Show more
  </button>
</div>
`;
